controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+const {sequelize} = require('../../databases/models');
+
+module.exports = async (req, res) => {
+  const {topics, locations, page} = req.query;
+  const result = [];
+
+  const limit = 7;
+  const offset = (page - 1) * limit;
+
+  try {
+    //TOPICS
+    if (topics) {
+      const topicsDetail = `SELECT * FROM topics WHERE topic_id IN (:topics)`;
+
+      const topicsDetailQueryResult = await await sequelize.query(topicsDetail, {
+        type: sequelize.QueryTypes.SELECT,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a redundant `await` keyword here which could lead to unexpected behavior. You only need one `await` when calling an asynchronous function.
```diff
- const topicsDetailQueryResult = await await sequelize.query(topicsDetail, {
+ const topicsDetailQueryResult = await sequelize.query(topicsDetail, {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->